tests/fault_tolerance/migration/utils.py

```diff
@@ -97,7 +109,7 @@ def send_request():
 
 def determine_request_receiving_worker(
     worker1: ManagedProcess, worker2: ManagedProcess, receiving_pattern: str
-) -> tuple:
+) -> tuple[ManagedProcess, str]:
```

Comment:
Maybe we don't need to be "clear" on the `tuple` composition, so `mypy` don't complain? It is because `mypy` is passing currently, I'm not sure why `mypy` starts to complain without changing any code in the function.
```suggestion
) -> tuple:
```